packages/@aws-cdk/core/lib/cfn-parse.ts

```diff
@@ -432,3 +470,5 @@ function specialCaseRefs(value: any): any {
 function undefinedIfAllValuesAreEmpty(object: object): object | undefined {
   return Object.values(object).some(v => v !== undefined) ? object : undefined;
 }
+
```

Comment:
Nope :)

Reply:
I have no idea how that line got there...I really don't really remember adding that, and I'm surprised the linter didn't catch the triple blank line.